appdriftctl.sh

```diff
@@ -42,7 +42,7 @@ case $response in
 esac
 
 # Authentication check
-if ! kubectl get namespaces &>/dev/null; then
+if ! kubectl auth can-i '*' deployments.apps -n somenamespace &>/dev/null; then
```

Comment:
Maybe check one of the provided namespaces? Move the `mapfile` statement up and use unique_namespaces[0]?

Reply:
Whoops, yeah, `somenamespace` was just an example. Should be replaced with a namespace in question, as it will probably give false when somenamespace doesn't exist